src/resources/views/event/edit.blade.php

```diff
@@ -22,138 +22,100 @@ class="bg-gray-200 hover:bg-gray-300 text-black font-bold py-2 px-4 rounded">Bac
                         <!--  Name SECTION  -->
                         <div class="px-4 py-5 bg-white sm:p-6">
                             @php($form_elem = "name")
-                            <label for="{{$form_elem}}" class="block font-medium text-md text-gray-700">Event's
-                                name</label>
-                            <input value="{{ old($form_elem, $event->{$form_elem}) }}" type="text" name="{{$form_elem}}"
-                                   id="{{$form_elem}}"
-                                   class="form-input rounded-md shadow-sm mt-1 block w-full"/>
-                            <small id="{{$form_elem}}Help" class="block font-medium text-sm text-gray-500">The name of
-                                the event</small>
-                            @error($form_elem)
-                            <p class="text-sm text-red-600">{{ $message }}</p>
-                            @enderror
+                            @livewire("forms.form", [
+                                'form_elem' => $form_elem,
+                                'type' => "text",
+                                'title' => "Event's name",
+                                'hint' => "The name of the event.",
+                                'previous' => $event->{$form_elem}
+                            ])
                         </div>
 
                         <!--  Event Type SECTION  -->
                         <div class="px-4 py-5 bg-white sm:p-6">
                             @php($form_elem = "event_type_id")
                             @php($list = ListEventType::list())
-                            <label for="{{$form_elem}}" class="block font-medium text-md text-gray-700">Event's
-                                Type</label>
-                            @livewire("select-dropdown", ['label' => $form_elem, 'placeholder' => "--
-                            Select the event type --", 'datas' => $list, 'selected_value' =>
-                            old($form_elem, $event->{$form_elem})])
-
-                            <small id="{{$form_elem}}Help" class="block font-medium text-sm text-gray-500">The type of
-                                the event</small>
-                            @error($form_elem)
-                            <p class="text-sm text-red-600">{{ $message }}</p>
-                            @enderror
+                            @livewire('forms.form', [
```

Comment:
Does the component allow you to use the search bar ?

Reply:
yes also